src/components/Tabs/PlotTab.tsx

```diff
@@ -51,13 +55,9 @@ export default function PlotTab(props: PlotTabProps): ReactElement {
               onChange={(event) => {
                 props.setFindTrackInputText(event.target.value);
               }}
+              onPressEnter={searchForTrack}
```

Comment:
what does this do if the field is empty?

Reply:
It will try to search for a `NaN` track, which won't exist. Nothing will happen on the UI, since the track is invalid.

I will actually implement a quick check to ignore `""` just because I can.